examples/centralized_threshold.rs

```diff
@@ -59,8 +60,21 @@ impl<H: Digest + FixedOutput> AlbaThresholdSignature<H> {
                 return None;
             }
 
-            // Collect the byte representation of valid signatures into a Vec
-            let prover_set: Vec<Data> = valid_signatures.keys().copied().collect();
+            let mut prover_set: Vec<Element<[u8; N]>> = Vec::with_capacity(valid_signatures.len());
+
+            prover_set.extend(valid_signatures.iter().filter_map(|(sig_bytes, index)| {
```

Comment:
Not sure what you are doing here either. Is N = DATA_LENGTH?

If so you could simply do:
```
let prover_set = valid_signatures.iter().map( |sig_bytes, i|
                Element {
                    data:sig_bytes,
                    index: Some(i)
                }
            ).collect();
```